app/page.tsx

```diff
@@ -5,8 +5,8 @@ import React from "react";
 
 export default function Home() {
   return (
-    <div className="flex justify-center items-start h-screen p-4">
```

Comment:
when you add h-screen here you make this div as big as the screen + the nav bar sits on top so it was doing some weird scrolling stuff